recipe/build.sh

```diff
@@ -1,18 +1,43 @@
 #!/bin/bash
 
+set -ex
+
 # Setup CMake build location
-mkdir build
-cd build
+mkdir build && cd build
 
 # Configure, build, test, and install.
-if [ "$(uname)" == "Linux" ];
-then
+if [ "$(uname)" == "Linux" ]; then
     # Stop Boost from using libquadmath.
-    export CXXFLAGS="${CXXFLAGS} -DBOOST_MATH_DISABLE_FLOAT128"
+    export CXXFLAGS="${CXXFLAGS} -D BOOST_MATH_DISABLE_FLOAT128"
+fi
+
+# fix issue with linker when using gcc 7.3.0
+if [[ ${target_platform} =~ .*linux.* ]]; then
+    export LDFLAGS="$LDFLAGS -Wl,-rpath-link,$PREFIX/lib"
+fi
+
+if [[ ${blas_impl} == openblas ]]; then
+    BLAS=open
+else
+    BLAS=mkl
 fi
-cmake -DCPU_ONLY=1 -DBLAS="open" -DCMAKE_INSTALL_PREFIX="${PREFIX}" -DCMAKE_INSTALL_LIBDIR=lib -Dpython_version=$PY_VER ..
-make
-make runtest
+
+cmake -D CPU_ONLY=1 \
+      -D BLAS="${BLAS}" \
+      -D CMAKE_INSTALL_PREFIX="${PREFIX}" \
+      -D NUMPY_INCLUDE_DIR="${SITE_PKGS}/numpy/core/include" \
```

Comment:
is that correct? or should it be ``SP_DIR`` instead of ``SITE_PKGS``?

Reply:
The conda-build env var is SP_DIR but this was working in the AR recipe :-/